parm/config/gfs/config.resources

```diff
@@ -331,7 +331,7 @@ case ${step} in
     export layout_x
     export layout_y
 
-    walltime="00:30:00"
+    walltime="00:50:00"
```

Comment:
What is causing this job to run this long?  Is this just running long on 1 machine or all?
Is this `$CASE` dependent?  If so, can we place a `case $CASE` block so that the lower resolutions do not have to request this much walltime?

Reply:
My guess is this is more case/ensemble number dependent.  I'll put this configuration with the C1152 case.